c2corg_api/views/user_mailinglists.py

```diff
@@ -0,0 +1,82 @@
+from c2corg_api import DBSession
+from c2corg_api.models.user import User
+from c2corg_api.models.mailinglist import Mailinglist
+from c2corg_api.views import cors_policy, restricted_json_view, \
+    restricted_view
+from c2corg_common.attributes import mailinglists
+from cornice.resource import resource
+
+
+def validate_mailinglist_statuses(request, **kwargs):
+    tested = []
+    for ml in request.validated:
```

Comment:
> If you are not using a schema or another validator that has written data to validated

I use no schema and no other validator. I have read that the standard ``colander_body_validator`` does nothing if no schema is provided so I have not added it to the list of validators of the post handler. Perhaps I should, in order to have ``request.validated`` in my validator? Or simply use ``request.POST`` (but that seems not very robust if we add another validator later?)?